packages/api/src/command/medical/patient/get-patient.ts

```diff
@@ -151,3 +159,75 @@ export const getPatientWithDependencies = async ({
   const organization = await getOrganizationOrFail({ cxId });
   return { patient, facilities, organization };
 };
+
+/**
+ * This function calculates the similarity between two patients using the Jaro-Winkler algorithm.
+ * It returns a score between 0 and 1, where 1 means the patients are identical. We calculate scores
+ * for the following fields: First Name, Last Name, Address Line 1, Address Line 2, City, State, Country, Zipcode.
+ * This function won't be called if gender and DOB are not identical, so that is a givem.
+ * @param patient1
+ * @param patient2
+ * @returns The average of the similarity scores for each field.
+ */
+export const calculatePatientSimilarity = (
```

Comment:
Should we move this to its own command/file? It seems pretty encapsulated in its own right and it would keep this file more manageable.

Reply:
Sure that makes sense to me!